pkg/reconciler/serverlessservice/serverlessservice.go

```diff
@@ -95,17 +95,15 @@ func (r *reconciler) Reconcile(ctx context.Context, key string) error {
 	reconcileErr := r.reconcile(ctx, sks)
 	if reconcileErr != nil {
 		r.Recorder.Eventf(sks, corev1.EventTypeWarning, "UpdateFailed", "InternalError: %v", reconcileErr.Error())
-		return reconcileErr
 	}
 	if !equality.Semantic.DeepEqual(sks.Status, original.Status) {
-		// Only update status if it changed.
```

Comment:
This comment still seems accurate?

Reply:
I think so. It's redundant, but relevant.